cfme/fixtures/v2v_fixtures.py

```diff
@@ -0,0 +1,522 @@
+        v2v_providers: vmware (and rhev in case of RHV migration ) , osp not needed.
+    """
+    provider_list = [v2v_providers.vmware_provider]
+    rhv_hosts = None
+
+    if v2v_providers.rhv_provider is not None:
+        rhv_hosts = v2v_providers.rhv_provider.hosts.all()
+        provider_list.append(v2v_providers.rhv_provider)
+
+    try:
+        for v2v_provider in provider_list:
+            hosts = v2v_provider.hosts.all()
+            for host in hosts:
+                host_data = [data for data in v2v_provider.data['hosts']
+                             if data['name'] == host.name]
+                if not host_data:
+                    pytest.skip("No host data")
+                host.update_credentials_rest(credentials=host_data[0]['credentials'])
+    except Exception:
+        logger.exception("Exception when trying to add the host credentials.")
+        pytest.skip("No data for hosts in providers, failed to retrieve hosts and add creds.")
+    # Configure conversion host for RHEV migration
+    if rhv_hosts is not None :
+        assign_conversion_tags_for_rhev(appliance, transformation_method, rhv_hosts)
+    if v2v_providers.osp_provider is not None:
+        set_conversion_instance_for_osp(appliance, transformation_method,
+                                        v2v_providers.osp_provider)
+
+
+def _tag_cleanup(host_obj, tag1, tag2):
+    """
+        Clean Up Tags
+
+        Returns: Boolean True if all Tags were removed/cleaned
+        or False means all required Tags are present on host.
+    """
+
+    def extract_tag(tag):
+        # Following strip will remove extra asterisk from tag assignment
+        return tag.category.display_name.strip(" *"), tag.display_name
+
+    valid_tags = {extract_tag(tag1), extract_tag(tag2)}
+    tags = host_obj.get_tags()
+    tags_set = set(map(extract_tag, tags))
+    # we always neeed 2 tags for migration, if total is less than 2
+    # don't bother checking what tag was it, just remove it and
+    # then add all required tags via add_tags() call. or if tags on host
+    # are not subset of valid tags, we still remove them.
+    if len(tags_set) < 2 or not tags_set.issubset(valid_tags):
+        host_obj.remove_tags(tags=tags)
+        return True
+    return False
+
+
+def assign_conversion_tags_for_rhev(appliance, transformation_method, rhev_hosts):
+    """Assigning tags to conversion host.
+       In 5.9 all rhev hosts and tagged with tags V2V - Transformation Host=t
+       and V2V - Transformation Method=vddk/ssh.These tags are automatically removed
+       once the provider is deleted.
+
+       In 5.10 rails console commands are run to configure all the rhev hosts.
+
+    Args:
+        appliance:
+        transformation_method : vddk or ssh as per test requirement
+        rhev_hosts: number of hosts in rhev to configure for conversion
+    """
+
+    tag1 = appliance.collections.categories.instantiate(
```

Comment:
@sshveta  we should only run tag thing for 5.9 since we don't need them on 5.10. Please specify it in else part of 5.10 or something like that.  This can save our 10s :dagger: